README.md

```diff
@@ -8,13 +8,27 @@ Free and open-source self-hosted url shortener service written in Go.
 
 Run in a terminal:
 
-```
-docker run -v curt-db:/data --name curt -p 19000:19000 -e PORT=19000 -e LOG_LEVEL=DEBUG -e API_KEY=your_apy_key -e HOST=http://localhost:19000/ salvatoreemilio/curt:latest
+```docker
+docker run -v curt-db:/data --name curt -p 19000:19000 -e PORT=19000 -e LOG_LEVEL=DEBUG -e API_KEY=your_api_key -e HOST=http://localhost:19000/ salvatoreemilio/curt:latest
 ```
 
+Or use a [docker-compose](./examples/compose.yaml) version 
 ### Examples
 
-- [compose.yaml](./examples/compose.yaml)
+- With an API Client send a **POST** request with this body
+    ```JSON
+    {
+        "url":"url_to_shorten"
+    }
+    ```
+    remember to insert API key in Header
```

Comment:
remember to insert an API key in the header if you configured it in the env variable

Reply:
ok, I correct it, review again